packages/styles/forms.css

```diff
@@ -187,7 +188,15 @@ textarea.Field--has-error:focus:hover,
   margin-bottom: 0;
   flex-direction: row-reverse;
   justify-content: flex-end;
-  flex-wrap: wrap;
+  flex-wrap: nowrap;
+}
+.column {
+  display: flex;
+  flex-direction: column;
+  width: fit-content;
+}
+.with-label {
+  flex-wrap: nowrap;
 }
```

Comment:
Cauldron styles are global meaning that it is extremely likely that these stills will conflict with already implemented styles in an app, especially given an ambiguous name like `.column` and doesn't really follow our [style guidelines](https://cauldron.dequelabs.com/style-guidelines).